README.md

```diff
@@ -22,6 +22,9 @@ spot.once("termination-cancelled", date => {
 spot.on("termination-not-detected", error => {
     console.log("termination-not-detected", error);
 });
+spot.on("instance-termination", error => {
+    console.log("termination-not-detected", error);
```

Comment:
can you update the text as "instance-termination" in example.

Reply:
Good spot :)